cobalt/browser/application.cc

```diff
@@ -212,9 +212,18 @@ GURL GetInitialURL(bool should_preload) {
   GURL initial_url = GURL(kDefaultURL);
   // Allow the user to override the default URL via a command line parameter.
   base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
-  if (command_line->HasSwitch(switches::kInitialURL)) {
+  bool has_initial_url_switch = command_line->HasSwitch(switches::kInitialURL);
+  const auto& argv = command_line->argv();
+  bool last_argv_is_not_a_switch =
+      (argv.size() > 1) && argv.back().substr(0, 2) != "--";
```

Comment:
Can't the substr call here throw an oob exception?

Reply:
No, not here; string::substr will only throw an exception if pos is out of range, which is zero here.

https://en.cppreference.com/w/cpp/string/basic_string/substr